src/Informer/CMakeLists.txt

```diff
@@ -13,8 +13,10 @@ add_spectre_library(${LIBRARY} ${LIBRARY_SOURCES})
 
 target_link_libraries(
   ${LIBRARY}
-  INTERFACE ErrorHandling
-  INTERFACE ${YAMLCPP_LIBRARIES}
+  PUBLIC
+    ErrorHandling
```

Comment:
For my own understanding, why is `ErrorHandling` public? Naively I would've expected `PRIVATE` since we don't pass classes defined in `ErrorHandling` around.

Reply:
Pushed a fixup with a correction. What do you think?